internal/standby/standby.go

```diff
@@ -138,6 +148,8 @@ func (s *Service) checkForOutage(currentTime time.Time) {
 	if err != nil {
 		s.publisher.PublishError("publishing current command", err)
 	}
+
+	s.logHandler.Append("Published command", currentInterval.LogFormat())
```

Comment:
Or error, if the previous condition evaluated to true.

Reply:
fixed by logging and returning inside the error clause